modules/module-mongodb/test/src/change_stream_utils.ts

```diff
@@ -147,14 +147,15 @@ export async function getClientCheckpoint(
   // Since we don't use LSNs anymore, the only way to get that is to wait.
 
   const timeout = options?.timeout ?? 50_000;
-  let lastCp: ActiveCheckpoint | null = null;
+  let lastCp: ReplicationCheckpoint | null = null;
 
   while (Date.now() - start < timeout) {
-    const cp = await bucketStorage.getActiveCheckpoint();
-    lastCp = cp;
-    if (!cp.hasSyncRules()) {
+    const storage = await storageFactory.getActiveStorage();
```

Comment:
Should this be `using storage`?

Reply:
Similar to my above comment, the lifecycle around disposing these is difficult. But in this specific case, disposing it would be wrong, since the same instance would still be cached for the next call.